### PR TITLE
 Reword the changelog fragments to better generate the CHANGELOG.md when we release 1.0.

### PR DESCRIPTION
**Pull request by jmrodri**
**Description of the change:**
Reword the changelog fragments to better generate the CHANGELOG.md when we release 1.0.

**Motivation for the change:**
Preparing for a 1.0 CHANGELOG.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: changelog/fragments/remove-pkg-predicate.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 entries:
   - description: >
-      Remove `pkg/predicate`: `ResourceFilterPredicate` is now available via
+      Removed `pkg/predicate`: `ResourceFilterPredicate` is now available via
       `pkg/ansible/predicate.NewResourceFilterPredicate()`; `GenerationChangedPredicate` has been renamed,
       modified, and moved (see this [operator-lib PR](https://github.com/operator-framework/operator-lib/pull/11)
       for details).
```
Comment (joelanford) [inline changelog/fragments/remove-pkg-predicate.yaml:6]: We're going to move `pkg/ansible/predicate` to `internal/pkg/ansible/predicate`, so maybe split this into two entries?

One that says:

"Removed `pkg/predicate.ResourceFilterPredicate`"
Migration: re-implement this in your project

And another that says how to use the moved/refactored predicates in operator-lib: 

Refactored and moved `pkg/predicate.GenerationChangedPredicate`
Migration: Use `crpredicate.Or(crpredicate.GenerationChangedPredicate{}, libpredicate.NoGenerationPredicate{})` (with some details about the imports)

File: changelog/fragments/rename-max-workers.yaml [hunk 2]
```diff
@@ -1,12 +1,14 @@
 entries:
   - description: >
-      **(Valid only for Ansible/Helm-based Operators)**. The flag `--max-workers` is no longer supported. 
-      Use `--max-concurrent-reconciles` instead.
+      **(Valid only for Ansible/Helm-based Operators)**. The `--max-workers` flag and `WORKERS_<Kind>_<Group>`
+      environment variable have been renamed to `--max-concurrent-reconciles` and
+      `MAX_CONCURRENT_RECONCILES_<Kind>_<Group>` respectively.
```
Comment (joelanford) [inline changelog/fragments/rename-max-workers.yaml:5]:
```suggestion
      The Ansible operator's `--max-workers` flag and `WORKERS_<Kind>_<Group>`
      environment variable have been renamed to `--max-concurrent-reconciles` and
      `MAX_CONCURRENT_RECONCILES_<Kind>_<Group>` respectively.
```

File: changelog/fragments/rm-legacy-gen-bundle.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove legacy generate bundle command
+      Removed legacy `generate bundle` command
```
Comment (joelanford) [inline changelog/fragments/rm-legacy-gen-bundle.yaml:5]:
```suggestion
      Removed `generate bundle` command support in legacy projects
```

File: changelog/fragments/rm-legacy-bundle-cmds.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove legacy "bundle create/validate" commands
+      Removed legacy `bundle create/validate` commands
```
Comment (joelanford) [inline changelog/fragments/rm-legacy-bundle-cmds.yaml:5]:
```suggestion
      Removed legacy `bundle create` command and removed `bundle validate` support in legacy projects
```

File: changelog/fragments/rm-legacy-gen-packagemanifests.yaml [hunk 5]
```diff
@@ -2,7 +2,7 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove legacy generate packagemanifests command
+      Removed legacy `generate packagemanifests` command
```
Comment (joelanford) [inline changelog/fragments/rm-legacy-gen-packagemanifests.yaml:5]:
```suggestion
      Removed `generate packagemanifests` command support in legacy projects
```

File: changelog/fragments/rm-legacy-go.yaml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 entries:
   - description: >
-      Removed legacy go CLI subcommands and features: `add`, `migrate`
+      Removed legacy Go CLI subcommands and features: `add`, `migrate`
       `print-deps`, `generate k8s`, `generate crds`, and `scorecard`.
```
Comment (joelanford) [inline changelog/fragments/rm-legacy-go.yaml:4]:
```suggestion
      Removed support in legacy Go projects for CLI subcommands and features: `add`, `migrate`
      `print-deps`, `generate k8s`, `generate crds`, and `scorecard`.
```

File: changelog/fragments/rm-legacy-run.yaml [hunk 7]
```diff
@@ -2,7 +2,7 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove legacy `run` and `cleanup` subcommands
+      Removed legacy `run` and `cleanup` subcommands
```
Comment (joelanford) [inline changelog/fragments/rm-legacy-run.yaml:5]:
```suggestion
      Removed `run` and `cleanup` subcommands support in legacy projects
```

File: changelog/fragments/scorecard-move-alpha.yaml [hunk 8]
```diff
@@ -1,7 +1,6 @@
 entries:
   - description: >
-      Changes the alpha scorecard command to become the standard scorecard 
-      command.  
+      The `alpha scorecard` comand has been promoted to the standard scorecard command.
```
Comment (joelanford) [inline changelog/fragments/scorecard-move-alpha.yaml:3]:
```suggestion
      The `alpha scorecard` command has been promoted to the standard scorecard command.
```

File: changelog/fragments/scorecard-parallel.yaml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 entries:
   - description: >
-      Changes the scorecard configuration format to include a new top-level
+      Changed the scorecard configuration format to include a new top-level
```
Comment (joelanford) [inline changelog/fragments/scorecard-parallel.yaml:3]: We should combine and updated this fragment with the scorecard component config PR fragment. Don't have to do that now though. I think there will be a bunch of this type of consolidation in the release prep PR.

File: changelog/fragments/use-operator-lib-metrics.yaml [hunk 10]
```diff
@@ -2,18 +2,11 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      Remove the implementation for `InstrumentedEnqueueRequestForObject`
-      handler from SDK repository and reference it from operator-lib instead.
+      Ansible & Helm-based Operators switched to using the `InstrumentedEnqueueRequestForObject` from
+      [operator-lib](https://github.com/operator-framework/operator-lib/blob/main/handler/instrumented_enqueue_object.go)
+      instead of controller-runtime's `EnqueueRequestForObject`.
```
Comment (joelanford) [inline changelog/fragments/use-operator-lib-metrics.yaml:7]: This should be consolidated with the removal of legacy metrics generation in Helm and Ansible operators fragments (again, in the release prep PR)

File: changelog/fragments/rm-cmd-legacy-ansible.yaml [hunk 11]
```diff
@@ -2,7 +2,7 @@
 # release notes and/or the migration guide
 entries:
   - description: >
-      The commands `operator-sdk new --type=ansible` and `operator-sdk add api` are no longer supported for Ansible-based Operators.
+      Removed legacy `new` and `add api` commands from Ansible-based Operators.
```
Comment (camilamacedo86) [inline changelog/fragments/rm-cmd-legacy-ansible.yaml:5]: All legacy removals IMO could be replaced for one fragment: 

```
# entries is a list of entries to include in
# release notes and/or the migration guide
entries:
  - description: >
      Remove support for the legacy layout. Operator SDK has a new CLI and project layout that is aligned with Kubebuilder. (//todo: add all pr's numbers)
      
    kind: "removal"
    # Is this a breaking change?
    breaking: yes
    # Migration can be defined to automatically add a section to
    # the migration guide. This is required for breaking changes.
    migration:
      header: Migrate your projects for the new layout
      body: >
        Operator SDK has a new CLI and project layout that is aligned with Kubebuilder.
        See `operator-sdk init -h` and the following doc on how to scaffold a new project:
        To migrate existing projects to the new layout see the migrations guides in
        https://sdk.operatorframework.io.
        - For Go-based Operators: https://master.sdk.operatorframework.io/docs/building-operators/golang/project_migration_guide/
        - For Helm-based Operators: https://master.sdk.operatorframework.io/docs/building-operators/helm/migration/
        - For Ansible-based Operators: TBD


```

File: changelog/fragments/helm-metrics-addr.yaml [hunk 12]
```diff
@@ -1,14 +1,14 @@
 entries:
   - description: >
-      Added `--metrics-addr` flag to helm operator to make it configurable, and
-      changed the default from `:8383` to `:8080`
+      Added `--metrics-addr` flag to the Helm operator to make it configurable, and
+      changed the default port from `:8383` to `:8080`
```
Comment (camilamacedo86) [inline changelog/fragments/helm-metrics-addr.yaml:4]: IMO we can combine the metrics change for Ansible and Helm: 


```
Breaking change: (Valid only for Ansible/Helm-based Operators). Added --metrics-addr flag to make it configurable, and changed the default port used by the metric endpoint binds to from :8383 to :8080. (#3489, #3440)
```

**Comment by jmrodri** [comment]
Closing this pull request in favor of the upcoming release PR.

**Pull request closed without merging**